src/zope/untrustedpython/builtins.py

```diff
@@ -45,35 +48,43 @@ def SafeBuiltins():
         'SystemExit', 'TabError', 'TypeError', 'UnboundLocalError',
         'UnicodeError', 'UserWarning', 'ValueError', 'Warning',
         'ZeroDivisionError',
-        '__debug__', '__name__', '__doc__', 'abs', 'apply', 'bool',
-        'buffer', 'callable', 'chr', 'classmethod', 'cmp', 'coerce',
-        'complex', 'copyright', 'credits', 'delattr',
+        'abs', 'bool',
+        'callable', 'chr', 'classmethod', 'cmp', 'coerce',
+        'complex', 'delattr',
         'dict', 'divmod', 'filter', 'float', 'frozenset', 'getattr',
         'hasattr', 'hash', 'hex', 'id', 'int', 'isinstance',
-        'issubclass', 'iter', 'len', 'license', 'list',
+        'issubclass', 'iter', 'len', 'list',
         'long', 'map', 'max', 'min', 'object', 'oct', 'ord', 'pow',
-        'property', 'quit', 'range', 'reduce', 'repr', 'reversed', 'round',
+        'property', 'range', 'reduce', 'repr', 'reversed', 'round',
         'set', 'setattr', 'slice', 'sorted', 'staticmethod', 'str', 'super',
-        'tuple', 'type', 'unichr', 'unicode', 'vars', 'xrange', 'zip',
+        'tuple', 'type', 'unichr', 'unicode', 'vars', 'zip',
         'True', 'False',
 
         # TODO: dir segfaults with a seg fault due to a bad tuple
         # check in merge_class_dict in object.c. The assert macro
         # seems to be doing the wrong think. Basically, if an object
         # has bases, then bases is assumed to be a tuple.
         # dir,
-    ]:
+    ]
+
+    if six.PY2:
+        safe_names.extend([
+            'apply',
+            'buffer',
+            'xrange',
+        ])  # pragma: PY2
```

Comment:
Should the changelog mention that some builtins (`license`, `copyright`, `credits`, `__name__`, `__doc__`, `__debug__`) are no longer available?